std_msgs/Header header

string element_name #UID for the body's world element.
geometry_msgs/Pose pose
